webapp/channels/src/components/admin_console/localization/localization.tsx

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import type {LocalizationSettings} from '@mattermost/types/config';
+
+import BooleanSetting from 'components/admin_console/boolean_setting';
+import DropdownSetting from 'components/admin_console/dropdown_setting';
+import MultiSelectSetting from 'components/admin_console/multiselect_settings';
+import {
+    AdminSection,
+    SectionContent,
+    SectionHeader,
+} from 'components/admin_console/system_properties/controls';
```

Comment:
I don't know if we should be reusing those here since they look like they're built specifically for that component. Perhaps this is something that could go into a settings group component